com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/Shaders/PathTracingBSDF.hlsl

```diff
@@ -110,7 +110,7 @@ bool SampleAnisoGGX(MaterialData mtlData,
     float VdotH;
     float3 localV, localH;
     float3x3 localToWorld = GetTangentFrame(mtlData);
-    SampleAnisoGGXVisibleNormal(inputSample, mtlData.V, localToWorld, roughnessX, roughnessY, localV, localH, VdotH);
+    SampleAnisoGGXVisibleNormal(inputSample.xy, mtlData.V, localToWorld, roughnessX, roughnessY, localV, localH, VdotH);
```

Comment:
@eturquin can you report those warning fix in a separate PR for path tracer? thanks

Reply:
Will do